### PR TITLE
docs: change capitalization for css hex color values

### PR DESCRIPTION
**Pull request by sreevani-ship-it**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

inconsistent capitalization of css hex color values.
Issue Number: 23113 https://github.com/angular/angular/issues/23113

## What is the new behavior?
changed hex color values from uppercase to lowercase and used 3 character hexadecimal notation where possible.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/examples/toh-pt5/src/app/app.component.css [hunk 1]
```diff
@@ -18,11 +18,11 @@ nav a {
   border-radius: 4px;
 }
 nav a:visited, a:link {
-  color: #607D8B;
+  color: #607d8b;
 }
 nav a:hover {
   color: #039be5;
-  background-color: #CFD8DC;
+  background-color: #cfd;
```
Comment (jenniferfell) [inline aio/content/examples/toh-pt5/src/app/app.component.css]: Did some characters get dropped accidentally? CFD8DC seems to be a medium gray, and CFD seems to be a bright green.

**Review by jenniferfell — CHANGES_REQUESTED** [review]

File: aio/content/examples/toh-pt5/src/app/app.component.css (hunk 1 above)
Comment (chembu) [reply aio/content/examples/toh-pt5/src/app/app.component.css]: I followed hex code style recommendation and reduced it to 3 digits. But on a second look, the RGB values are different when it is 3 digits. Reverted it back to 6 digit code.

Comment (gkalpak) [reply aio/content/examples/toh-pt5/src/app/app.component.css]: 3-digit hex color values are equivalent to 6-digit color values with each digit duplicated. I.e. `#cfd` is the same as `#ccffdd`, which is indeed quit different than `#cfd8dc` (in 6-digit hex color values each digit pair corresponds to one of R, G, B).

**Review by jenniferfell — APPROVED** [review]

**Comment by jenniferfell** [comment]
@IgorMinar On the associated issue, @jelbourn comments that the Google style is lowercase.

File: aio/content/examples/toh-pt5/src/app/app.component.css [hunk 2]
```diff
@@ -18,11 +18,11 @@ nav a {
   border-radius: 4px;
 }
 nav a:visited, a:link {
-  color: #607D8B;
+  color: #607d8b;
 }
 nav a:hover {
   color: #039be5;
-  background-color: #CFD8DC;
+  background-color:  #cfd8dc;
```
Comment (alfaproject) [inline aio/content/examples/toh-pt5/src/app/app.component.css]: Double space after colon

**Review by gkalpak — APPROVED** [review]
I haven't checked if there are more, but the changes on this PR lgtm.

**Review by IgorMinar — APPROVED** [review]
I fixed the extra ws issue and pushed to this PR. lgtm now.

**Pull request closed without merging**